### PR TITLE
instancetype: Move to v1beta1

### PR DESCRIPTION
**Pull request by lyarwood**
Moves all instance type documentation to the new v1beta1 version.

### CODE REVIEW

File: docs/virtual_machines/instancetypes.md [hunk 1]
```diff
@@ -467,12 +467,26 @@ EOF
...
+
+* This version captured complete `VirtualMachine{Instancetype,ClusterInstancetype,Preference,ClusterPreference}` objects within the created `ControllerRevisions`
+
+* This version is backwardly compatible with `instancetype.kubevirt.io/v1alpha1`.
+### `instancetype.kubevirt.io/v1beta1`
+
+* The following instance type attribute has been added:
+  * [`Spec.Memory.OvercommitPercent`](https://kubevirt.io/api-reference/main/definitions.html#_v1beta1_memoryinstancetype)
+
+* The following preference attributes have been added:
```
Comment (0xFelix) [inline docs/virtual_machines/instancetypes.md:484]: What about preference requirements?

Comment (lyarwood) [reply docs/virtual_machines/instancetypes.md:484]: I'll add this now.

**Comment by lyarwood** [comment]
/hold cancel

`v1.0.0` has been released, I'll add in the missing preference requirements details now.

File: docs/virtual_machines/instancetypes.md [hunk 2]
```diff
@@ -28,10 +29,10 @@ spec:
...
+KubeVirt provides two Instancetype based [`CRDs`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/), a cluster wide [`VirtualMachineClusterInstancetype`](https://kubevirt.io/api-reference/main/definitions.html#_v1beta1_virtualmachineclusterinstancetype) and a namespaced [`VirtualMachineInstancetype`](https://kubevirt.io/api-reference/main/definitions.html#_v1beta1_virtualmachineinstancetype). These `CRDs` encapsulate the following resource related characteristics of a `VirtualMachine` through a shared [`VirtualMachineInstancetypeSpec`](https://kubevirt.io/api-reference/main/definitions.html#_v1beta1_virtualmachineinstancetypespec):
```
Comment (0xFelix) [inline docs/virtual_machines/instancetypes.md:32]: `KubeVirt provides two Instancetype based` can you rephrase this?

**Review by 0xFelix — APPROVED** [review]
/lgtm

Thanks!

**Comment by lyarwood** [comment]
/cc @cwilkers 

Would you mind taking a look at this when you have time?

**Comment by cwilkers** [comment]
Everything looks in order, congrats on the API promotion to beta

/approve
